src/app/docs/docs.component.spec.ts

```diff
@@ -34,15 +36,15 @@ describe('DocsComponent', () => {
     })
   );
 
-  // beforeEach(() => {
-  //   fixture = TestBed.createComponent(DocsComponent);
-  //   component = fixture.componentInstance;
-  //   fixture.detectChanges();
-  // });
+  beforeEach(() => {
+    fixture = TestBed.createComponent(DocsComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
 
-  // it('should create', () => {
-  //   expect(component).toBeTruthy();
-  // });
+  it('should create', () => {
```

Comment:
This was commented out; in Angular 15 the tests would throw an error for no tests in the spec file.